lib/teselagen/api/tests/test_test_client.py

```diff
@@ -258,10 +267,13 @@ def test_create_experiment(self, experiment: List[Dict[str, Any]]) -> None:
         assert "id" in response.keys()
         assert response["id"] is not None
 
-    def test_delete_experiment(self, logged_client: TESTClient,
-                               experiment: List[Dict[str, Any]]) -> None:
-        client = logged_client
-        experiment_id = int(experiment["id"])
+    def test_delete_experiment(
```

Comment:
I assume that the endpoint verifies the action and returns a "not OK" code if the experiment could not be deleted - @AndresPerezTesela, could you please confirm this information?

Otherwise, here we would only be testing the interface of the `delete_experiment` method, since it always returns `None` unless an exception is raised by the request - we could verify that the experiment is not present using the `get_experiments` method but that would entangle / mess up the tests, which would not be nice.

Reply:
The endpoint return the ID of the experiments if everything goes well, or an http error (depending on the error) otherwise